src/client/mod.rs

```diff
@@ -903,11 +987,12 @@ where
     #[cfg(feature = "vault_0_6_2")]
```

Comment:
Should this cfg flag come out?

Reply:
Thanks for finding that! I must have only searched for `0_6_1`.
I found one other place too.